lib/valkyrie/persistence/solr/model_converter.rb

```diff
@@ -40,11 +40,22 @@ def to_h
         "id": id,
         "join_id_ssi": "id-#{id}",
         "created_at_dtsi": created_at
-      }.merge(attribute_hash)
+      }.merge(add_single_values(attribute_hash))
     end
 
     private
 
+      def add_single_values(attribute_hash)
+        attribute_hash.select do |k, v|
+          field = k.to_s.split("_").last
+          property = k.to_s.gsub("_#{field}", "")
+          next true if field.include?("m")
```

Comment:
There's nothing in the schema now that would cause problems, but I wonder if a more conservative check would be better here, e.g., `field.end_with?("m") || field.end_with?("mv")`

Reply:
:+1: Should perform better too. Added